docker/Dockerfile

```diff
@@ -33,7 +33,6 @@ WORKDIR /sgl-workspace
 
 RUN pip3 --no-cache-dir install --upgrade pip \
     && pip3 --no-cache-dir install "sglang[all]" \
-    && pip3 --no-cache-dir uninstall -y triton triton-nightly \
```

Comment:
Why is `uninstall` being deleted here?

Reply:
We do not need triton nightly anymore, I will delete the logic.